pkg/object/object.go

```diff
@@ -45,9 +48,34 @@ type OpenCompose struct {
 	Volumes  []Volume
 }
 
+func validateReplicas(replicas *int32) error {
+	if replicas != nil && *replicas < 0 {
+		return fmt.Errorf("Replica count is negative")
+	}
+	return nil
+}
+
+func (s *Service) Validate() error {
+	// validate service name, like it cannot have underscores, etc.
+
+	// validate containers
+
+	// validate replicas
+	if err := validateReplicas(s.Replicas); err != nil {
```

Comment:
Even I thought this might be overdoing but then there arent other tests so just for testing replicas I will need to create entire stucts object.

And here I can do specific tests like these one: https://github.com/redhat-developer/opencompose/pull/66/files#diff-4f9edbaf8553598b5a269f232bab233cR16